src/main/java/org/example/filetypes/CsvFile.java

```diff
@@ -0,0 +1,48 @@
+package org.example.filetypes;
+
+import com.opencsv.CSVWriter;
+import com.opencsv.bean.*;
+import org.example.fileData.FileData;
+import java.io.*;
+import java.util.List;
+
+public class CsvFile extends FileData {
```

Comment:
CsvFile sugeruje jakby to miał być plik jakiś. Bardziej odpowiednią nazwą byłby CsvFileManager a klasa którą rozszerza Abstract/BaseFileManager